core/vm/gas_table.go

```diff
@@ -100,24 +98,12 @@ var (
 
 func gasExtCodeSize(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	usedGas := uint64(0)
-	slot := stack.Back(0)
-	address := slot.Bytes20()
-	if evm.chainRules.IsPrague {
-		usedGas += evm.TxContext.Accesses.TouchAddressOnReadAndComputeGas(address[:], uint256.Int{}, trieUtils.CodeSizeLeafKey)
-	}
-
```

Comment:
I remove the gas accounting stuff in this file, because all verkle-specific gas is moved to `operations_verkle.go`. This makes it less of a rebasing nightmare.